bioptim/interfaces/acados_interface.py

```diff
@@ -626,9 +626,8 @@ def configure(self, options: dict):
         options: dict
             The dictionary of options
         """
-        if not options:
-            return
-
+        if options is None:
+            options = []
```

Comment:
I disagree with this, because each time you call acados the option down there are overwritten. The default behavior should be that is passes once if empty then if recalled when empty, it returns without overriding the previous setup

Reply:
They are not thanks to :

`if self.ocp_solver is None:`

Which was introduced for receding horizon applications, not to overwrite these options (which cannot be overwritten after compilation anyway)

Reply:
When I tested it few weeks ago, it was overriding the parameters... Can you design a test to actually check if the parameters can be modified once and are indeed not modified the second time?

Reply:
Once the problem is compiled, there are only a few sets of parameters that can be changed before calling solve() again.
They are listed [here](https://github.com/pyomeca/bioptim/blob/41391b5d2c6b8376ef464356a6a72feed7d47a5b/bioptim/interfaces/acados_interface.py#L654).
It must be noted that all changes occurring after compilation are invisible to the user (the .json file is left unchanged, and there are no getter implemented to check for the changes).
We must trust acados devs on this point, see [this discussion](https://discourse.acados.org/t/getting-costs-constraints-yref-etc-after-solver-creation/309) for more info.